vectordatasource/meta/sql.py

```diff
@@ -167,6 +167,8 @@
     'mz_get_min_zoom_highway_level_gate': 'mz_get_min_zoom_highway_level_gate',
     'util.calculate_path_major_route':    'mz_calculate_path_major_route',
     'mz_calculate_ferry_level':           'mz_calculate_ferry_level',
+    'util.is_building':                   'mz_calculate_is_building_or_part',
+    'util.tag_yes_to_bool':               'notimplemented',
```

Comment:
@zerebubuth is it important for sql functions to exist for all new functions that we add?